DigitalLearningSolutions.Web/ViewModels/Common/CentreInfoViewModel.cs

```diff
@@ -0,0 +1,19 @@
+namespace DigitalLearningSolutions.Web.ViewModels.Common
+{
+    using System.Collections.Generic;
+
+    public class CentreInfoViewModel
+    {
+        public CentreInfoViewModel(int centreId, string bannerText)
+        {
+            Routes = new Dictionary<string, string>
+            {
+                { "centreId", centreId.ToString() }
+            };
+            BannerText = bannerText;
+        }
+
+        public Dictionary<string, string> Routes { get; set; }
+        public string BannerText { get; set; }

```

Comment:
I think BannerText is optional in the DB at the moment, so we should only render the bannerText if it exists, otherwise just show the "More information" text. 
Also, can we post a pic of this in the external channel so HEE can come up with alternative wording (I think they requested it at the Show&Tell but I might misremember)

Reply:
Nope you're right, they did ask - I'll do that.